test/e2e_node/pod_resize_test.go

```diff
@@ -101,38 +103,46 @@ func supportsInPlacePodVerticalScaling(ctx context.Context, f *framework.Framewo
 	return false
 }
 
-func getTestResourceInfo(tcInfo TestContainerInfo) (v1.ResourceRequirements, v1.ResourceList, []v1.ContainerResizePolicy) {
-	var res v1.ResourceRequirements
+func getTestResourceInfo(tcInfo TestContainerInfo) (v1.ResourceRequirements, v1.ResourceRequirements, v1.ResourceList, []v1.ContainerResizePolicy) {
+	var res, statRes v1.ResourceRequirements
 	var alloc v1.ResourceList
 	var resizePol []v1.ContainerResizePolicy
 
-	if tcInfo.Resources != nil {
+	createResourceRequirements := func(resources *ContainerResources) v1.ResourceRequirements {
```

Comment:
I refactored pod resize e2e tests in https://github.com/kubernetes/kubernetes/pull/128143. Is it possible to update the test file?